Libraries/Components/Keyboard/Keyboard.js

```diff
@@ -13,5 +13,10 @@
 
 const NativeEventEmitter = require('NativeEventEmitter');
 const KeyboardObserver = require('NativeModules').KeyboardObserver;
+var dismissKeyboard = require('dismissKeyboard');
 
-module.exports = new NativeEventEmitter(KeyboardObserver);
+let keyboard = new NativeEventEmitter(KeyboardObserver);
```

Comment:
Would that be correct?  I'm adding a function to `keyboard`.  Would that be defeating the idea of `const`?